boards/shields/wnc_m14a2a/wnc_m14a2a.overlay

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Foundries.io
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	/*
```

Comment:
This needs to be discussed as I think a change in our DTS processing is needed for:
1) To remove the warning generated by these aliases using labels with a "gpios" portion in their name
OR
2) Perhaps we can make a change which would allow the actual GPIO setting in the modem DTS to be assigned a label:
mdm-boot-mode-sel-gpios = &arduino_d1;
This would make these hacky aliases go away.

Reply:
I'll try to discuss this at connect with dts guys.